app/models/job.rb

```diff
@@ -40,6 +40,13 @@ def state_counts
   scope :queued, -> { where(state: "queued") }
   scope :running, -> { where(state: ["preparing", "running", "analyzing"] ) }
   delegate :queue_name, to: :job_group
+  
+  # Filter jobs for today's SLO targets
+  scope :core_hours_today, -> { day = 1.day.ago;
```

Comment:
Yep, although I think this is a bit of a BBC-specific implementation here. Maybe something like: slo_core_period ?